client/extensions/woocommerce/app/settings/email/email-settings/index.js

```diff
@@ -1,23 +1,224 @@
+	},
+	{
+		field: 'email_cancelled_order',
+		title: translate( 'Cancelled order' ),
+		subtitle: translate( 'Sent when a new order is marked \'cancelled\'.' ),
+	},
+	{
+		field: 'email_failed_order',
+		title: translate( 'Failed order' ),
+		subtitle: translate( 'Sent when a new order is marked \'failed\'.' ),
+	},
+];
+
+// id enabled
+const customerNotifications = [
+	{
+		field: 'email_customer_on_hold_order',
+		title: translate( 'Order pending payment' ),
+		subtitle: translate( 'Sent when an order is marked \'payment pending\'.' ),
+	},
+	{
+		field: 'email_customer_processing_order',
+		title: translate( 'Processing order' ),
+		subtitle: translate( 'Sent when an order is marked \'payment processing\'.' ),
+	},
+	{
+		field: 'email_customer_completed_order',
+		title: translate( 'Completed order' ),
+		subtitle: translate( 'Sent when an order is marked \'paid in full\'.' ),
+	},
+	{
+		field: 'email_customer_refunded_order',
+		title: translate( 'Refunded order' ),
+		subtitle: translate( 'Sent when an order is marked \'payment refunded\'.' ),
+	},
+	{
+		field: 'email_customer_new_account',
+		title: translate( 'Refunded order' ),
+		subtitle: translate( 'Sent when an order is marked \'payment refunded\'.' ),
+	},
+];
 
 class Settings extends React.Component {
 
 	constructor( props ) {
 		super( props );
 	}
+
+	fetchSettings = ( props ) => {
+		const { siteId, fetchSettings } = props;
+		siteId && fetchSettings( siteId );
+	}
+
+	componentDidMount = () => {
+		this.fetchSettings( this.props );
+	};
+
+	componentWillReceiveProps = newProps => {
+		if ( newProps.siteId !== this.props.siteId ) {
+			this.fetchSettings( newProps );
+		}
+	};
+
+	notificationsToggle = () => {
+		undefined;
+	}
+
+	recipientsChange = () => {
+		undefined;
+	}
+
 	/* eslint-disable wpcalypso/jsx-classname-namespace */
 	render() {
+		const { loading, loaded, settings } = this.props;
+		const waiting = ! loaded || loading;
+
 		return (
 			<div className="email-settings__container">
+				<div className="email-settings__origin">
+					<Card className="email-settings__origin-title">
+						Origin
+					</Card>
+					<Card className="email-settings__origin-settings">
+						{ waiting && <p className="email-settings__loading" /> }
+						{ loaded && <NotificationsOrigin
+								item={ fromName }
+								recipient={ settings.email[ fromName.field ] }
+								onChange={ this.recipientsChange }
+							/> }
+						{ waiting && <p className="email-settings__loading" /> }
+						{ loaded && <NotificationsOrigin
+								item={ fromAddress }
+								recipient={ settings.email[ fromAddress.field ] }
+								onChange={ this.recipientsChange }
+							/> }
+					</Card>
+				</div>
+				<div className="email-settings__internal-notifications">
+					<Card className="email-settings__internal-notifications-title">
+						<div className="email-settings__section-title-text">
+							{ translate( 'Internal notifications' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **50** times:
`translate( 'notifications' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/email-settings-ui)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).